app/views/layouts/application.html.erb

```diff
@@ -63,7 +63,7 @@
       <%= render 'shared/govuk_logo' %>
       <%= yield :proposition_header %>
     </div>
-    <%= render 'shared/search_banner' if TradeTariffFrontend.search_banner? %>
+    <%= render 'shared/search_banner' if TradeTariffFrontend.search_banner? && @search %>
```

Comment:
If we want to be safe, I guess we should use:
`defined?(@search) && @search.present?`

Should not a problem with the current Ruby MRI version 
`@search` returns nill in case of:
- not declared
- it contains nil

but things could change

Reply:
Ok, ignore my comment 😸 
we actually commonly use instance variables, which might not be even declared, in the views.